stdlib/public/runtime/EnvironmentVariables.cpp

```diff
@@ -187,9 +191,32 @@ extern "C" char **_environ;
 #endif
 #endif
 
+#if defined(__ANDROID__)
+// On Android, also try loading runtime debug env variables from system props.
+static void platformInitialize(void *context) {
+  (void)context;
+#define SYSPROP_PREFIX "debug.swift.runtime."
+#define VARIABLE(name, type, defaultValue, help)                          \
+  do {                                                                    \
+    char name##_string[PROP_VALUE_MAX] = "";                              \
+    if (__system_property_get(SYSPROP_PREFIX #name, name##_string) > 0) { \
+      swift::runtime::environment::name##_isSet_variable = true;          \
+      swift::runtime::environment::name##_variable =                      \
+        parse_##type(#name, name##_string, defaultValue);                 \
+    }                                                                     \
+  } while (0);
```

Comment:
```suggestion
  {                                                                       \
    char name##_string[PROP_VALUE_MAX] = "";                              \
    if (__system_property_get(SYSPROP_PREFIX #name, name##_string)) {     \
      swift::runtime::environment::name##_isSet_variable = true;          \
      swift::runtime::environment::name##_variable =                      \
          parse_##type(#name, name##_string, defaultValue);               \
    }                                                                     \
  }
```

Reply:
Oh, and I can also use a single property value string buffer variable rather than one per variable. Let me make that change, too.